core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -286,6 +277,7 @@ func (i *pluginOracleCreator) createFactoryAndTransmitter(
 	destFromAccounts []string,
 	publicConfig ocr3confighelper.PublicConfig,
 	destChainFamily string,
```

Comment:
nit: Now that you are passing offrampAddrStr, I think the previous param destChainFamily is not needed?